namespaces/blueprint-parse-result.md

```diff
@@ -0,0 +1,119 @@
+# Blueprint Parser Result Namespace
+
+This document defines elements needed for holding the result of parsing
+API Blueprint. This document uses [API Namespace][].
```

Comment:
Questions:

* Is this going to be specific to API Blueprint?
* Will there be any overlap between parsing other API Description formats?